### ISSUE TITLE
Reading mpt or mpr files with comma as decimal

### ISSUE THREAD

**Issue by NilsBuett** [vetschn/eclabfiles#17]
Hallo Nicolas, 

I want to read mpt or mpr Files 'to_df' but due to the german system we are working on our decimal separators are comma. Is there a way to replace all comma with dots during import. 

Example:
df = eclabfiles.to_df(filename)

My df type for a comma value is a tuple. For example I have got a value of 2,5V in my df the value is 2,5 but as [2, 5]. Do I need to combine all tuples and convert them to float?

**Comment by PeterKraus** [comment]
Hello Nils, we've added locale support to the `mpt` parsers in the maintained version of this extractor, see [yadg](github.com/dgbowl/yadg).

Long story short, if you're extracting a `mpt` file on the same PC (i.e. the locale of the file and of your PC is consistent), it should just work, using:

``` python
from yadg.extractors import extract

dataset = extract(filetype="eclab.mpt", path="your_file.mpt")
```

Note that this returns an `xarray.Dataset`, not a `pandas.DataFrame`, so you have to have `xarray` installed. If you're extracting on a different machine and the locales might be different, please open an issue in the `yadg` repo and we can assist you.

**Comment by NilsBuett** [comment]
I'll give it a try and if it doesn't work, I'll get back to you.

**Comment by NilsBuett** [comment]
Okay I have opened an issue in the yadg repo, cause it is not working.

**Comment by PeterKraus** [comment]
Fixed in https://github.com/dgbowl/yadg/pull/152, available in `yadg-5.1`.
